assets/src/components/Tooltip.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react'
```

Comment:
Old assignment planning change. I just did what worked.

Reply:
This change was made due since the event is passed as one of the arguments and cannot be access as `d3.event`

More on this https://observablehq.com/@d3/d3v6-migration-guide#events 

This change applies for all assignment and resource accessed view